com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -1229,10 +1194,6 @@ public virtual void WriteDiscreteActionMask(IDiscreteActionMask actionMask)
             {
                 m_ActionMasker = new DiscreteActionMasker(actionMask);
             }
-            // Disable deprecation warnings so we can call the legacy overload.
```

Comment:
I think this can be empty now (and maybe we can get rid of `m_ActionMasker` too?)

Reply:
And maybe get rid of the DiscreteActionMasker class entirely?